protocol/service.go

```diff
@@ -31,6 +31,16 @@ func Version(req *VersionRequest) *VersionResponse {
 	return DefaultService.Version(req)
 }
 
+// SupportedLanguages uses DefaultService to process the given SupportedLanguagesRequest to get the supported drivers.
+//proteus:generate
+func SupportedLanguages(req *SupportedLanguagesRequest) *SupportedLanguagesResponse {
```

Comment:
`ListDrivers` might be a better name because we could potentially have multiple drivers for a single language (different versions/runtimes/forks).

Reply:
Agree.

Reply:
`ListDrivers` was my first option, but @mcuadros and @smola suggested `SupportedLanguages` because reasons.
I can rename it again if you agree :)

Reply:
Let's see what the bosses say.

Reply:
It actually depends on the protocol level. For internal protocol `SupportedLanguages` makes sense. But for bblfshd `ListDrivers` is the only possible name (because of reasons mentioned above).

Reply:
We had a discussion about what belongs to the administration API and what belongs to the public user API. I think we all agreed that listing supported languages is useful for end-users. We already have 4 very different use cases where this is good: bblfsh-playground, engine, gitbase, ml. Other aspects of driver install might or might not be needed.

@mcuadros also commented that public API should be the same as API for individual drivers standalone, but I think it's better that he elaborates that himself.

Reply:
I think that this API should cover most of the fields found in the driver manifest (although we can start by having only a minimal set of fields, like in current PR).

And I probably know the reason why Maximo wants an API to be exactly the same - it will allow using bblfshctl to send requests to individual drivers without additional flags to switch the protocol and without running bblfshd.

Reply:
After reading @smola then I think it could stay as it is and if in the future a driver supports more than one language we could add it to the returned SupportedLanguages here pointing to the same DriverDetails (or duplicating it) of the language supporting both.